content/nginxaas-azure/app-protect/configure-waf.md

```diff
@@ -101,7 +101,33 @@ The following table shows the path to the precompiled policy file that needs to
 
 To view the contents of the available security policies, navigate to the azure portal and select the **Security Policies** tab in the App Protect section.
 
-{{<note>}}Custom policies are not supported at this time.{{</note>}}
+## Custom Policies
+
+NGINXaas for Azure also supports custom security policies. You can create and modify custom security policies to deploy to NGINX App Protect Instances using the API or Azure Portal.
+
+### Manage Custom Policies
+
+To create a custom security policy in the Azure Portal:
+
+1. Select your deployment
+2. Select **NGINX app protect WAF** from the menu on the left
+3. Select **Custom Policies**
+4. Select **Add Custom Security Policy** to open the policy editor
+
+In the policy editor, enter the **Name**, **File path**, your policy content, and then select **Save**. The **File path** is optional and will default to the path "/etc/app_protect/conf/" plus the policy **Name** with a ".json" extension. For more information on policy configuration and syntax, refer to the NGINX App Protect [configuration guide](https://docs.nginx.com/nginx-app-protect-waf/v5/configuration-guide/configuration/).
+
+   {{< img src="nginxaas-azure/custom-policy-editor.png" alt="NGINXaaS Custom Policy Editor." >}}
```

Comment:
```suggestion
   {{< img src="nginxaas-azure/custom-policy-editor.png" alt="NGINXaaS Custom Policy Editor." >}}
```

What does this screenshot add that is not already covered by the instructions above?

Please review the guidelines for screenshot. 
https://github.com/nginx/documentation/blob/main/templates/style-guide.md#guidelines-for-screenshots

Please note that we strongly discourage the use of screenshots unless strictly necessary. Added screenshots need to include a descriptive alt text for screen readers and improving accessibility.

Reply:
Agreed, removed.